app/models/katello/pool.rb

```diff
@@ -2,14 +2,22 @@ module Katello
   class Pool < Katello::Model
     include Katello::Authorization::Pool
 
-    attr_accessor :quantity_attached
+    def quantity_attached
+      if activation_keys.present?
+        activation_keys.first.get_key_pools
+          .select { |pool| pool['id'] == cp_id }.first.try(:[], 'amount')
+      end
+    end
```

Comment:
@johnpmitsch Would it make sense for a `Pool` to return the sum of `quantity_attached` from all its activation keys in a method called `Pool#quantity_attached`?